src/routes.ts

```diff
@@ -112,6 +114,22 @@ export const routes: RouteItemObj[] = [
     connected: false,
     onlyDropdown: true,
   },
+  {
+    name: 'IPFSApp',
+    href: 'https://app.ens.eth.limo/',
+    label: 'navigation.ipfsApp',
+    disabled: false,
+    connected: false,
+    onlyDropdown: true,
+  },
+  {
+    name: 'Feedback',
+    href: '',
+    label: 'navigation.feedback',
+    disabled: false,
+    connected: false,
+    onlyDropdown: true,
+  },
```

Comment:
nit but can we reorder these so "IPFS ENS App" and "Old ENS App" are on the same line and lower down?